Content.Shared/Shuttles/Components/IFFComponent.cs

```diff
@@ -38,13 +38,18 @@ public enum IFFFlags : byte
     /// <summary>
     /// Should the label for this grid be hidden at all ranges.
     /// </summary>
-    HideLabel,
+    HideLabel = 1,
 
     /// <summary>
     /// Should the grid hide entirely (AKA full stealth).
     /// Will also hide the label if that is not set.
     /// </summary>
-    Hide,
+    Hide = 2,
+
+    /// <summary>
+    /// Is this a player shuttle
+    /// </summary>
+    IsPlayerShuttle = 4,
```

Comment:
does this get overwritten if a player uses an IFF console? we may need a different flag/tag or just an empty component instead, if so

Reply:
I have tested with both IFF consoles, the IsPlayerShuttle flag is preserved, and I don't see any code that should affect it.
Could store it in a separate variable, but the IFFFlag had lots of bits to spare